kola/tests/crio/crio.go

```diff
@@ -142,14 +143,28 @@ var crioContainerTemplate = `{
 				]
 			}
 		}
-	}
+	},
+	"mounts": [{
+		"container_path": "/tmp/test",
+		"host_path": "/tmp/test",
+		"read_only": false,
+		"selinux_relabel": true,
+		"propagation": 2
+	}]
 }`
 
 // RHCOS has the crio service disabled by default, so use Ignition to enable it
 var enableCrioIgn = conf.Ignition(`{
   "ignition": {
     "version": "2.2.0"
   },
+  "storage": {
+	"directories": [{
+		"filesystem": "root",
+		"path": "/tmp/test",
+		"mode": 511
```

Comment:
Does this need to be `0777`?

Reply:
It probably doesn't HAVE to be `0777` ... `0755` would probably be fine as well.

Reply:
I'm okay with it either way. Feel free to change or just merge